blocks/library/latest-posts/index.php

```diff
@@ -74,6 +78,9 @@ function register_block_core_latest_posts() {
 			'categories'      => array(
 				'type' => 'string',
 			),
+			'className'       => array(
```

Comment:
The introduction of this attribute should have been complemented by an update to fixtures, which auto-generates from server attributes. Running `npm run fixtures:regenerate` on master introduces an unexpected change.